CHANGELOG.md

```diff
@@ -7,4 +7,4 @@
 * Add running jobs components to display temp static data. Refs UIDEXP-6.
 * Implement logic for initiating the export job process once the user has selected file. Refs UIDEXP-20.
 * Add validation to QueryFileUploaderComponent to let user upload only file with csv extension. UIDEXP-21.
-* Updated `stripes` to `v3.0.0`, `stripes-core` to `4.0.0` and `react-intl` to `2.9.0`. UIDEXP-30.
+* Update `stripes` to `v3.0.0`, `stripes-core` to `4.0.0` and `react-intl` to `2.9.0`. UIDEXP-30.
```

Comment:
it seems the right descripiton is missed

Reply:
I decided not to add a description for these chores as these changes are implementation details of UIDEXP-20, the description for which already exists in the changelog.